app/views/teams/_teams_table.html.erb

```diff
@@ -20,30 +20,24 @@
         <td><%= team.mentor ? team.mentor.user.user_name : 'Not assigned' %></td>
         <td>
           <%if team.get_team_submission_status(team.get_own_submissions_in_order[1]) == "Submitted" %>
-          <!--<a class="btn btn-primary btn-xs" onClick="openModalBox(<%#= team.id %>)" href="../submissions/#modal_<%#=team.id %>"> Submission Link </a>-->
-            <a class="btn btn-primary btn-xs" href="<%#=team.get_own_submissions_in_order[1].video_link %>"> Video Link </a>
-            <br>
-            <a class="btn btn-primary btn-xs" href="<%#=team.get_own_submissions_in_order[1].poster_link %>"> Poster Link </a>
+            <% submission = team.get_own_submissions_in_order[1] %>
+          <a class="btn btn-primary btn-s" href="<%= milestone_team_submission_path(submission.milestone_id, submission.team_id, submission.id)%>"> View Submission </a>
           <% else %>
             <%= team.get_team_submission_status(team.get_own_submissions_in_order[1]) %>
           <% end %>
         </td>
         <td>
           <%if team.get_team_submission_status(team.get_own_submissions_in_order[2]) == "Submitted" %>
-            <!--<a class="btn btn-primary btn-xs" onClick="openModalBox(<%#= team.id %>)" href="../submissions/#modal_<%#=team.id %>"> Submission Link </a>-->
-            <a class="btn btn-primary btn-xs" href="<%#=team.get_own_submissions_in_order[2].video_link %>"> Video Link </a>
-            <br>
-            <a class="btn btn-primary btn-xs" href="<%#=team.get_own_submissions_in_order[2].poster_link %>"> Poster Link </a>
+            <% submission = team.get_own_submissions_in_order[2] %>
+            <a class="btn btn-primary btn-s" href="<%= milestone_team_submission_path(submission.milestone_id, submission.team_id, submission.id)%>"> View Submission </a>
           <% else %>
             <%= team.get_team_submission_status(team.get_own_submissions_in_order[3]) %>
```

Comment:
Think there is a typo bug on line 34. It should be team.get_own_submissions_in_order[2] instead of 3

Reply:
Thanks for spotting the bug @Wklee96. To remove the redundancy, I have extracted it into a for loop in #712. I have incorporated the change here: 	https://github.com/nusskylab/nusskylab/blob/c826c66acffa5d45bba566a0fcfd3e731c0c895e/app/views/teams/_teams_table.html.erb#L29